core/domain/user_services.py

```diff
@@ -23,10 +23,10 @@
 import re
 
 from core.platform import models
-current_user_services = models.Registry.import_current_user_services()
-(user_models,) = models.Registry.import_models([models.NAMES.user])
 import feconf
 import utils
+current_user_services = models.Registry.import_current_user_services()
```

Comment:
These should be above feconf, similar to how this file looked like before it was changed. Within a group, imports are alphabetized.